src/OfxNet/Sgml/SgmlParser.cs

```diff
@@ -32,15 +34,39 @@ public SgmlElement Parse(string path, Encoding encoding)
             var line = reader.ReadLine();
             ++this.lineNumber;
 
-            if (string.IsNullOrWhiteSpace(line) == false)
+            if (!string.IsNullOrEmpty(line))
             {
-                this.ProcessLine(line);
+                var tags = line.Split("<").ToList();
+                tags.ForEach(t =>
+                {
+                    if (!string.IsNullOrWhiteSpace(t))
+                    {
+                        this.ProcessLine(Wellformed(t));
+                    }
+                });
             }
         }
 
         return this.root;
     }
 
+    private static string Wellformed(string tag)
+    {
+        if (tag.Contains('<', StringComparison.InvariantCultureIgnoreCase) &&
```

Comment:
```suggestion
        if (tag.Contains('<', StringComparison.Ordinal) &&
```
Or `OrdinalIgnoreCase`